config/zones/IN-NO.yaml

```diff
@@ -20,7 +20,17 @@ contributors:
   - pierresegonne
   - sahitya-pavurala
   - willbeaufoy
+  - mathilde-daugy
 timezone: Asia/Kolkata
+emissionFactors:
+  direct:
+    unknown:
+      source: CEA 2022; assumes 56% biomass, 36% hydro, 8% unknown
+      value: 45
+  lifecycle:
+    unknown:
+      source: CEA 2022; assumes 56% biomass, 36% hydro, 8% unknown
+      value: 192
```

Comment:
We should also add isRenewable and isLowCarbon values as well so we get the "correct" renewable and low carbon share.